public/js/interface.js

```diff
@@ -0,0 +1,116 @@
+var calculateTotal = function(){
+  $.get("/total", function(data) {
+    $("#totalPrice").text("Total: £" + data.total);
+    $("#tax").text("Tax (8.64%): £" + data.tax);
+    $("#afterTax").text("After Tax: £" + data.after);
+  });
+};
+
+var addToList = function(){
+  $.get("/items", function(data) {
+     calculateTotal();
+     $("#errorTill").empty();  
+     $("#tillNumbers").empty();
+    var size = +data.item.length;
+    for (var i = 0; i < size; i++) {
+      var button = data.item[i] + " x" + data.quantity[i] + "= £" +
+       (+data.price[i]).toFixed(2) + " " + '<button value="' +
+       data.item[i] +'" class="x '+ data.item[i].replace(/\s+/g, '') +
+       '" id="' + data.quantity[i] +'">x</button>';
+       $('<div />',{html: button}).appendTo("#tillNumbers");
+    }
+  });   
+};
+
+var validate = function(item, quantity) {
+  var count = $("#tillNumbers").children().length;
+  if (+quantity !== 0 && count < 7) {
```

Comment:
Magic numbers here! A validate object would help as you can then set properties with all of the values you're looking for